nipy/io/tests/test_image_io.py

```diff
@@ -45,7 +45,7 @@ def test_badfile():
     # nibabel prior 2.1.0 was throwing a ImageFileError for the not-recognized
     # file type.  >=2.1.0 give a FileNotFoundError.
     try:
-        from nibabel.py3k import FileNotFoundError
+        from numpy.compat.py3k import FileNotFoundError
     except ImportError:
         FileNotFoundError = IOError
```

Comment:
Should just remove this. `FileNotFoundError` is now a builtin.

Reply:
I assume this package is no longer attempting to support Python 2.7?

Reply:
Given that there's hardly maintenance effort to keep it running, extending what little there is to 2.7 seems silly. I don't call the shots here, so I might get overruled, but it seems safe to drop Python 2.

Reply:
Yup - no need to maintain Python 2.7 at this stage.